examples/analog_in/cont_voltage_acq_int_clk_every_n_samples_event.py

```diff
@@ -0,0 +1,37 @@
+﻿"""Example of analog input voltage acquisition with events.
+
+This example demonstrates how to use Every N Samples events to
+acquire a continuous amount of data using the DAQ device's
+internal clock. The Every N Samples events indicate when data is
+available from DAQmx.
+"""
+
+import pprint
+import time
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+pp = pprint.PrettyPrinter(indent=4, compact=True)
+
+with nidaqmx.Task() as task:
+
+    def callback(task_handle, every_n_samples_event_type, number_of_samples, callback_data):
+        """Callback function for reading singals."""
+        print("Every N Samples callback invoked.")
+        pp.pprint(task.read(number_of_samples_per_channel=number_of_samples))
+
+        return 0
+
+    task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+    task.timing.cfg_samp_clk_timing(
+        1000.0, sample_mode=AcquisitionType.CONTINUOUS, samps_per_chan=50
+    )
+    task.register_every_n_samples_acquired_into_buffer_event(50, callback)
```

Comment:
The samps_per_chan parameter on cfg_samp_clk_timing specifies the total number of samples to acquire for finite and a "hint" to the buffer size for continuous. It is completely unrelated to the register_every_n_samples_acquired_into_buffer_event parameter, which specifies how many samples to acquire **per event**.

(1000 S/s) / (50 S/event) = 20 event/s, which is still too fast.